packages/cli/README.md

```diff
@@ -128,7 +134,7 @@ jobs:
       - run: npx code-pushup autorun --upload.apiKey=${{ secrets.PORTAL_API_KEY }}
 ```
 
-## CLI commands and options
+## 🤖 CLI commands and options
```

Comment:
I'd swap the emojis - _CI automation_ should be 🤖, and _CLI commands and options_ should be 🛠.

Reply:
done